frontend/src/i18n/locales/en/common.json

```
@@ -1,19 +1,18 @@
 {
-  "Are you sure ... ?": "Are you sure ... ?",
-  "No data to be shown.": "No data to be shown.",
-  "Open menu": "Open menu",
-  "Other": "Other",
-  "Pick a timezone": "Pick a timezone",
-  "Start typing to search a timezone": "Start typing to search a timezone",
-  "Timezone": "Timezone",
-  "breadcrumbs": "breadcrumbs",
-  "Must be greather than or equal to x": "Must be greather than or equal to {{number}}",
-  "Must be less than x characters": "Must be less than {{number}} characters",
-  "Must be a positive number": "Must be a positive number",
-  "Must be a valid hash (less than x characters)": "Must be a valid hash (less than {{number}} characters)",
-  "Must be an integer number": "Must be an integer number",
-  "Must enter a valid filename (less than x characters)": "Must enter a valid filename (less than {{number}} characters)",
-  "Must be a reverse domain ID like io.example.MyApp": "Must be a reverse domain ID like io.example.MyApp",
-  "Must be less than x characters": "Must be less than {{number}} characters",
-  "something_wrong": "Something went wrong. Check the form or try again later..."
+  "confirmation_prompt": "Are you sure ... ?",
+  "no_data_message": "No data to be shown.",
+  "open_menu": "Open menu",
+  "other_option": "Other",
+  "pick_timezone": "Pick a timezone",
+  "search_timezone_prompt": "Start typing to search a timezone",
+  "timezone_label": "Timezone",
+  "breadcrumbs_label": "breadcrumbs",
+  "min_value_error": "Must be greater than or equal to {{number}}",
+  "max_length_error": "Must be less than {{number}} characters",
+  "positive_number_error": "Must be a positive number",
+  "valid_hash_error": "Must be a valid hash (less than {{number}} characters)",
+  "integer_number_error": "Must be an integer number",
+  "valid_filename_error": "Must enter a valid filename (less than {{number}} characters)",
+  "reverse_domain_id_error": "Must be a reverse domain ID like io.example.MyApp",
+  "generic_error": "Something went wrong. Check the form or try again later..."
```

Same here about ellipsis.
```suggestion
  "generic_error": "Something went wrong. Check the form or try again later…"
```